compute.py

```diff
@@ -42,20 +45,27 @@ def compute(cls, query: Word, secret: Word):
             greenIdxSet = secretIdxSet.intersection(queryIdxSet)
             for greenIdx in greenIdxSet:
                 filter[greenIdx] = Color.GREEN
+<<<<<<< HEAD
```

Comment:
conflict markers?

Reply:
Oh I guess I missed this because you pushed to main right around when I made my branch